sass/typography/_typography.scss

```diff
@@ -2,7 +2,7 @@ html {
 	-moz-osx-font-smoothing: grayscale;
 	-webkit-font-smoothing: antialiased;
 	font-smoothing: antialiased;
-	font-size: 16px;
+	font-size: 100%;
```

Comment:
disagree. tested earlier version (`16px`) and it does not override the browser setting (Firefox & Chrome). are you actually seeing this issue?

Reply:
nevermind, was testing in an environment that was overriding the 16px oops. it's fine.